docs/reference/commandline/trust_signer_remove.md

```diff
@@ -0,0 +1,150 @@
+---
+title: "signer-remove"
+description: "The signer-remove command description and usage"
+keywords: "signer, notary, trust"
+---
+
+<!-- This file is maintained within the docker/cli Github
+     repository at https://github.com/docker/cli/. Make all
+     pull requests against that repo. If you see this file in
+     another repository, consider it read-only there, as it will
+     periodically be overwritten by the definitive file. Pull
+     requests which include edits to this file in other repositories
+     will be rejected.
+-->
+
+# trust signer-remove
+
+```markdown
+Usage:  docker trust signer-remove [OPTIONS] signer IMAGE [IMAGE...]
+
+Remove a signer from an image
```

Comment:
Unrelated side note: Perhaps we should reword this description when defining the subcommand as this can be used to remove a signer from one or more images.

Reply:
agreed - maybe `Remove a signer from one or more images` ?   Same applies to `signer-add`